src/main/java/frc/robot/subsystems/CoralManipulator.java

```diff
@@ -12,58 +12,83 @@
 import edu.wpi.first.units.measure.Angle;
 import edu.wpi.first.wpilibj.DigitalInput;
 import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Commands;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import edu.wpi.first.wpilibj2.command.WaitUntilCommand;
 import frc.robot.Constants;
-import frc.util.control.nt.PIDTuning;
+import frc.robot.util.control.nt.PIDTuning;
 
 import java.util.function.BooleanSupplier;
 
 public class CoralManipulator extends SubsystemBase {
-    private final TalonFX pivotMotor;
+    // private final TalonFX pivotMotor;
+    // private final DoublePublisher anglePub;
+    // private final MotionMagicVoltage motionMagic;
+    // private final PIDTuning pidTuning;
+    // private boolean openLoop;
+    // private boolean changedPos;
+    // private Angle targetAngle;
+
```

Comment:
just delete the unused code. if we need to add it back we can look at the previous version of the code